src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -340,7 +340,8 @@
  *
  * <blockquote><pre><strong>{@code
  * // Print out a number using the localized number, integer, currency,
- * // and percent format for each locale}</strong>{@code
```

Comment:
This comment is originally in strong font within the code example. Replacing both instances of `@ code` with `@ snippet` will just create two separate snippet blocks, one for the comment and one for the code which does not seem ideal.

To maintain the strong font, the comment has to be outside of the code snippet.

If that's not desirable, I think some alternatives are

- Leave the comment in the code snippet and get rid of the strong font 
- Keep the strong font but convert the comment into a normal text description

Reply:
I would leave the comment in the code snippet.  I don't see a compelling reason for it to be  as it was originally

Reply:
+1